services/provider/client/client.go

```diff
@@ -134,16 +134,29 @@ func (cc *OCSProviderClient) AcknowledgeOnboarding(ctx context.Context, consumer
 	return cc.Client.AcknowledgeOnboarding(apiCtx, req)
 }
 
-func (cc *OCSProviderClient) FulfillStorageClassClaim(ctx context.Context, consumerUUID, storageClassClaimName, encryptionMethod string, storageType pb.FulfillStorageClassClaimRequest_StorageType) (*pb.FulfillStorageClassClaimResponse, error) {
+type StorageType uint
+
+const (
+	Blockpool StorageType = iota
+	Sharedfilesystem
```

Comment:
The names `Blockpool` and `Sharedfilesystem` are very common throughout the codebase can we prefix them to prevent naming issues?

Maybe `StorageTypeBlockpool` and `StorageTypeSharedfilesystem`

Reply:
Done